workspaces/topology/packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -14,7 +14,9 @@
  * limitations under the License.
  */
 import React from 'react';
-import { Button, Grid } from '@material-ui/core';
+import Button from '@mui/material/Button';
+import Grid from '@mui/material/Unstable_Grid2';
```

Comment:
This one I'm less sure what we should do. It's marked as stable in MUI v6. For the Demo site I lift it as is.

Reply:
This is what worked in my Demo site PR: `import Grid from '@mui/material/Grid';` 🤔

Reply:
Thanks for the information. Somehow when I worked on it, it showed "deprecated" for `Grid` and I referred this [page](https://mui.com/material-ui/migration/migration-grid-v2/#with-material-ui-v5) to update the import with `Unstable_Grid2`, now it magically compiled with `Grid`... Changing it back to `Grid` 🙌